js/table/tableview.ts

```diff
@@ -2884,7 +2885,7 @@ export class TableView {
             if (p.PlayerId() === playerId) {
                 const saveMask = 16;
                 p.Status(status | (p.Status() & saveMask));
-                if (playerId === authManager.loginId()) {
+                if (playerId === self.authInformation.loginId()) {
```

Comment:
Don’t add any new reference to `self`. Use `this` and change functions to arrow style